frontend/README.md

```diff
@@ -46,8 +46,15 @@ You will now have the backend API available on `localhost:8000/docs` and the fro
 Note that changes to the frontend React code will be visible live while developing due to the bind mount in the docker-compose file (see line 37). It is not necessary to rebuild the images to view the frontend changes.
 
 To see what containers are running, use `docker ps`. To see all containers, including stopped containers, add option `-a`. You can remove a particular container using `docker rm <container>`. Another handy command is `docker logs <containerName>` to view a container's output.
+#### Running against live backend
```

Comment:
will probs clean up the frontend readme at some point but thanks for the update!